source/Firebase/Functions/Enums.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using ObjCRuntime;
+
+namespace Firebase.Functions
+{
+    [Native]
+    public enum FunctionsErrorCode : long
+    {
+        FunctionsErrorCodeOK = 0,
```

Comment:
Remove `FunctionsErrorCode` from enum values, as this would be repetitive with the enum name

```suggestion
        Ok = 0,
```